src/components/TextField/TextField.tsx

```diff
@@ -124,6 +126,7 @@ const TextField: VibeComponent<TextFieldProps, unknown> & {
       trim = false,
       role = "",
       required = false,
+      requiredErrorText = DEFAULT_REQUIRED_ERROR_TEXT,
```

Comment:
I think it should be customizable using the validations object

Reply:
But as a user, you can't know if the text field is empty and required. This is also a specific use case. The logic is held inside the component itself